src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -127,7 +127,11 @@ when friends send you messages.`,
         ListHeaderComponent={
           <>
             <View style={{height: 52}}>
-              <Search animatedValue={0} onPress={navigateToContactScreen} />
+              <Search
+                animatedValue={0}
+                onPress={navigateToContactScreen}
+                isAnon={selectedTab === 1}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnon` prop is being set based on whether the `selectedTab` is 1. This could potentially lead to a bug if the tab order changes in the future or if more tabs are added. It would be better to explicitly check for the 'Anonymous' tab rather than relying on its position.

```diff
-                isAnon={selectedTab === 1}
+                isAnon={selectedTabName === 'Anonymous'}
```
In this change, `selectedTabName` should be the actual name of the selected tab. You'll need to ensure that you have a mechanism in place to get the name of the selected tab.


<!-- This is an auto-generated comment by OSS CodeRabbit -->